### ISSUE TITLE
[DL3025] triggered with spaces inside [] in CMD or ENTRYPOINT

### ISSUE THREAD

**Issue by 4x0v7** [hadolint/hadolint#240]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Rule is not triggered

### Actual behavior
Rule is triggered

### Steps to reproduce the behavior

Consider the following Dockerfile:
```
FROM alpine:3.7
ENTRYPOINT [ "something" ]
```
Removing the space between `something"` and `]` will pass (DL3025 not triggered)
This happens also for `CMD`

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

Haskell Dockerfile Linter v1.10.2-2-g63fe7fb

```bash
/dev/stdin:2 DL3025 Use arguments JSON notation for CMD and ENTRYPOINT arguments
```

**Dockerfile** (if relevant)

```Dockerfile
FROM alpine:3.7
ENTRYPOINT [ "something" ]

```

**Additional environment details (OS, stack version, etc.)**
Docker version 18.03.1-ce, build 9ee9f40

**Comment by lorenzo** [comment]
thanks for reporting, we'll fix this soon
